apps/prairielearn/src/pages/enroll/enroll.html.ts

```diff
@@ -23,10 +23,6 @@ export function Enroll({
   courseInstances: CourseInstance[];
   resLocals: Record<string, any>;
 }) {
-  // Temporary for testing.
-  courseInstances.forEach((ci) => {
-    ci.instructor_access = false;
-  });
 
```

Comment:
Removing an extraneous blank line. I'd recommend running `make link`/`make format` to catch/fix formatting issues locally: https://prairielearn.readthedocs.io/en/latest/dev-guide/#coding-style

```suggestion
```